packages/webviz-core/package.json

```diff
@@ -10,10 +10,10 @@
     "@simonwep/pickr": "1.4.7",
     "@trust/webcrypto": "0.9.2",
     "antd": "3.26.5",
-    "chart.js": "2.7.3",
+    "chart.js": "davidswinegar/Chart.js#ab170d249db7cae3313d9e2515ef35283c9bb5e8",
     "chartjs-plugin-annotation": "0.5.7",
-    "chartjs-plugin-datalabels": "0.4.0",
-    "chartjs-plugin-zoom": "0.6.6",
+    "chartjs-plugin-datalabels": "0.7.0",
```

Comment:
looking at this and realizing this is out of date - we should build a check to make sure this is synchronized internally.